packages/usbmux/usbmux.0.9/opam

```diff
@@ -0,0 +1,48 @@
+opam-version: "1.2"
+maintainer: "Edgar Aroutiounian <edgar.factorial@gmail.com>"
+authors: "Edgar Aroutiounian <edgar.factorial@gmail.com>"
+homepage: "https://github.com/onlinemediagroup/ocaml-usbmux"
+bug-reports: "https://github.com/onlinemediagroup/ocaml-usbmux/issues"
+license: "BSD-3"
+dev-repo: "https://github.com/onlinemediagroup/ocaml-usbmux.git"
+build: [
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
+  ["ocaml" "setup.ml" "-build"]
+]
+install: ["ocaml" "setup.ml" "-install"]
+build-test: [
+  ["ocaml" "setup.ml" "-configure" "--enable-tests"]
+  ["ocaml" "setup.ml" "-build"]
+  ["ocaml" "setup.ml" "-test"]
+]
+remove: ["ocaml" "%{etc}%/usbmux/_oasis_remove_.ml" "%{etc}%/usbmux"]
+depends: [
+  "ANSITerminal"
+  "cmdliner" {build}
```

Comment:
On Tue, 26 Jan 2016 02:23:24 -0800, Daniel Bünzli wrote:
> 
> > +  "cmdliner" {build}
> 
> Also it's not a good idea to declare libraries as {build} dependencies. You are preventing users from getting bugfix releases.

oasis2opam makes that decision because cmdliner is the dependency of an executable only.  That decision is questioned in the code for the very reason you mention above.  On the other hand, for an executable, one may argue that its behavior has been tested — and dependencies issues have been worked around — and that an update of the library may cause undesired behavior...